README.md

```diff
@@ -1,62 +1,65 @@
+- [12-counters.elm](https://ellie-app.com/3Z99K3tpYbKa1)
+- [13-input-box.elm](https://ellie-app.com/3Z99YBq3jM8a1)
+- [14-todos.elm](https://ellie-app.com/3Z9bnpxFvffa1)
+- [15-todos.elm](https://ellie-app.com/3Z9bCnDPRv8a1)
+- [16-todos.elm](https://ellie-app.com/3Z9bXz3RsyQa1)
+- [17-todos.elm](https://ellie-app.com/3Z9cf8ZrCpta1)
+- [18-editable-todos.elm](https://ellie-app.com/3Z9cDJ3N9g4a1)
+- [19-editable-todos.elm](https://ellie-app.com/3Z9d3dXRdPxa1)
+- [20-editable-todos.elm](https://ellie-app.com/3Z9dhJHkXg6a1)
+- [21-localstorage-editable-todos.elm](https://ellie-app.com/3Z9dKFRMyNDa1)
+- [22-localstorage-editable-todos.elm](https://ellie-app.com/3Z9fg9YtbV4a1)
+- [23-filter-todos.elm](https://ellie-app.com/3Z9fFVqF9Pra1)
+- [24-filter-todos.elm](https://ellie-app.com/3Z9gnSZKYPTa1)
+- [25-navigation-todos.elm](https://ellie-app.com/3wR7QgwzngSa1/25)
+
+## How to run and view examples 21-25 on your computer
+You can use the Ellie links for examples 21-25, but if you want to run it locally,
+you can copy/paste the following commands which will clone the Github repo,
+then compile example 21 to a JavaScript file called elm.js,
+then you will open the HTML for example 21 in your default browser.
+
+```bash
 git clone https://github.com/bryanjenningz/25-elm-examples.git
 cd 25-elm-examples
-elm-make 21-localstorage-editable-todos.elm --output elm.js --yes
+elm make 21-localstorage-editable-todos.elm --output elm.js
 open 21-localstorage-editable-todos.html
-  </code>
-</pre>
-<p>The above code will compile and open example 21 in your default browser. To compile and view examples 22 through 25 locally, you do the same process of compiling the Elm file you want to the elm.js JavaScript file, then opening the HTML file you want to view in your browser.</p>
-
-<h2>Other Free Resources</h2>
-<ul>
-  <li><a href="https://guide.elm-lang.org/">Official Elm Guide</a></li>
-  <li><a href="https://ellie-app.com/">Ellie App</a></li>
-  <li><a href="https://www.reddit.com/r/elm/">Elm subreddit</a></li>
-  <li><a href="https://elmlang.slack.com/">Elm Slack channel</a></li>
-  <li><a href="https://github.com/rtfeldman/elm-spa-example">RealWorld Elm example app</a></li>
-</ul>
-
-<p>These 25 Elm examples were inspired by the <a href="http://elm-lang.org/examples">examples</a> on the Elm website and the <a href="https://guide.elm-lang.org/">official Elm guide</a>. Each example tries to build off of the previous example by adding a small amount of code so that it's easy to understand and see how you can build stuff in Elm. If you have any suggestions for more examples that should be added or if anything is unclear, add an issue or make a pull request.</p>
+```
+
+The above code will compile and open example 21 in your default browser.
+To compile and view examples 22 through 25 locally, you do the same process of
+compiling the Elm file you want to the elm.js JavaScript file,
+then opening the HTML file you want to view in your browser.
```

Comment:
Line 49 can probably be replaced with this:
then running `elm reactor` and selecting the HTML file in your browser at localhost:8000.